src/frontend/pages/recruitment/Configuration/ConfigurationPage.js

```diff
@@ -86,20 +93,87 @@ const TimeContainer = styled.div`
 `;
 
 const ConfigurationPage = () => {
-  const [firstDeadline, setFirstDeadline] = React.useState(new Date());
-  const [secondDeadline, setSecondDeadline] = React.useState(new Date());
-  const [orientationDate, setOrientationDate] = React.useState(new Date());
-  const [confirmDeadline, setConfirmDeadline] = React.useState(new Date());
-  const [startTime, setStartTime] = React.useState(new Date());
-  const [endTime, setEndTime] = React.useState(new Date());
+  const { configuration, updateConfiguration } = useConfiguration();
+
+  const [interviewMeetingLink, setInterviewMeetingLink] = useState();
+  const [interviewFirstRoundDeadline, setInterviewFirstRoundDeadline] =
+    useState(new Date());
+  const [interviewSecondRoundDeadline, setInterviewSecondRoundDeadline] =
+    useState(new Date());
+  const [newMembersMeetingLink, setNewMembersMeetingLink] = useState();
+  const [newMembersMeetingDate, setNewMembersMeetingDate] = useState(
+    new Date(),
+  );
+  const [newMembersMeetingStartTime, setNewMembersMeetingStartTime] = useState(
+    new Date(),
+  );
+  const [newMembersMeetingEndTime, setNewMembersMeetingEndTime] = useState(
+    new Date(),
+  );
+  const [newMembersFormLink, setNewMembersFormLink] = useState();
+  const [newMembersFormDeadline, setNewMembersFormDeadline] = useState(
+    new Date(),
+  );
```

Comment:
It's better to combine all these useState hooks into a single one that stores the entire configuration as a JSON object, then reference it using JSON notation (just so you don't have to define state setter functions for each individual field).